### PR TITLE
fromfunction

### LINKED ISSUE #11995: fromfunction
Working on (#8431-> #17806)

### PR DESCRIPTION
**Pull request by Sarvesh-Kesharwani**
Closed #11995

### CODE REVIEW

File: ivy/functional/frontends/jax/numpy/logic.py [hunk 1]
```diff
@@ -284,3 +284,31 @@ def packbits(x, /, *, axis=None, bitorder="big"):
     bits = ivy.expand_dims(bits, axis=tuple(range(x.ndim - 1)))
     packed = (x << bits).sum(axis=-1).astype("uint8")
     return ivy.swapaxes(packed, axis, -1)
+
+
+@with_supported_dtypes(
```
Comment (abdulasiraj) [inline ivy/functional/frontends/jax/numpy/logic.py:289]: I would like to share a quick consideration: I'm curious about the potential rationale behind the differentiation in supported data types for the `fromfunction` and `setxor1d` functions compared to other functions that appear to accommodate a wider range of data types. Is there a particular justification for this distinction that I might be overlooking? Your insights on this matter would be greatly appreciated.

**Review by abdulasiraj — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/jax/numpy/logic.py (hunk 1 above)
Comment (Sarvesh-Kesharwani) [reply ivy/functional/frontends/jax/numpy/logic.py:289]: @abdul, you are absolutly correct, the supported wrapper is not needed anymore after i modified the func, thanks.
will push the changes it soon.

and will test setxor1d again as well.

**Review by abdulasiraj — CHANGES_REQUESTED** [review]
I think, you forgot to update the datatypes on `fromfunction`.

File: ivy/functional/frontends/jax/numpy/logic.py [hunk 2]
```diff
@@ -283,3 +283,28 @@ def packbits(x, /, *, axis=None, bitorder="big"):
     bits = ivy.expand_dims(bits, axis=tuple(range(x.ndim - 1)))
     packed = (x << bits).sum(axis=-1).astype("uint8")
     return ivy.swapaxes(packed, axis, -1)
+
+
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes({"0.4.14 and below": ("bool",)}, "jax")
```
Comment (abdulasiraj) [inline ivy/functional/frontends/jax/numpy/logic.py:289]: Hi @Sarvesh-Kesharwani 
you're again restricting the dtypes, I think we talked about this already.

**Review by abdulasiraj — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/jax/numpy/logic.py (hunk 2 above)
Comment (Sarvesh-Kesharwani) [reply ivy/functional/frontends/jax/numpy/logic.py:289]: Hi @abdul, about the fromfunction https://github.com/unifyai/ivy/pull/21432,

although you are correct that i should not restrict this func for bool dtype but upon testing i found that my implementation which internally uses ivy's gather_nd function which doesn't support bfloat16 dtype, tell me what you think? 
image

**Comment by abdulasiraj** [comment]
Hi @Sarvesh-Kesharwani 
can you please resolve the conflicts?
Thanks!

**Review by abdulasiraj — CHANGES_REQUESTED** [review]
LGTM!

**Pull request merged**